WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -373,9 +373,13 @@ def time_until_competition
   end
 
   def dues_formatting
+    if !Money.default_bank.rates_updated_at || Money.default_bank.rates_updated_at < Time.now - 1.days
```

Comment:
I like that! :smile: 
In Ruby, instead of calculating `Time.now - 1.days` you can do `1.days.ago`. But please check this, I'm writing this off the top of my head

Reply:
However, locally `rates_updated_at` does not change after `update_rates` is called. And `rates_updated_at` gives year-month-date while Time.now gives year-month-date hour:minutes:seconds, can they still be compared?

I still added this change because `update_rates` only took 1-2 extra seconds to run on my machine 😬

Reply:
Just passing by and I was curious about that PR; if updating the rates is some kind of routine jobs, I'd recommend putting this in a rake task instead of doing this in a controller method.

We do similar stuff [here](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/lib/tasks/work.rake) for instance, and it wouldn't be too hard to add an extra job.

Reply:
Also, just a minor nit: Rails has a lot of convenience features for working with date and time. I believe you can do something like `1.days.ago` instead of `Time.now - 1.days`.

Reply:
Would I still need to use this though if I just put the currency update into the rake jobs